internal/pkg/plugin/binary.go

```diff
@@ -120,23 +120,56 @@ func Uninstall(name, libexecdir string) error {
 
 	meta, err := loadMetaByName(name, pluginDir)
 	if err != nil {
-		// figure out if this is a not found error?
 		return err
 	}
 
 	sylog.Debugf("Found plugin %q, meta=%#v", name, meta)
+
 	return meta.uninstall()
 }
 
-func loadMetaByName(name, plugindir string) (*Meta, error) {
-	fh, err := os.Open(metaPath(plugindir, name))
+// Enable enables the plugin named "name" found under "libexecdir"
+func Enable(name, libexecdir string) error {
```

Comment:
It does seem like these functions here are more like "application code" rather than the underlying backing package. I think that the `internal/app/singularity/remote_*` files are good examples of this. I think we revisit this in follow up PRs as we flesh out the API we're exposing here in `internal/pkg/plugin`

Reply:
Ok